tests/integration/contracts/blockifier_contracts/feature_contracts/cairo0/test_proxy.cairo

```diff
@@ -0,0 +1,112 @@
+%lang starknet
+
+from starkware.cairo.common.cairo_builtins import HashBuiltin
+from starkware.starknet.common.syscalls import library_call, library_call_l1_handler
+from starkware.cairo.common.math import assert_not_zero
+
```

Comment:
If there's any context you can add about this file (where it came from, why we have it, etc.) it would be good to add here.